### DISCUSSION TITLE
Pinch to change grid size in app.

### DISCUSSION THREAD

**Discussion by haritvsmurali** [ente-io/ente#4086] [Enhancements]
Pinch with two fingers to zoom in, zoom out and change grid size of thumbnails in the app.

**Comment by libbyslayer** [comment]
Currently only Date wise grouping is available in the app in main screen. Want Month Wise Grouping as well while zooming out and Date Wise while zooming in.

**Comment by Yrlish** [comment]
This is one feature really I miss from Google photos, being able to resize the photo grid by zooming.

I know I can change grid size between 2-6 in settings. But having it changeable eaily by zooming would be amazing.

**Comment by jordanbertasso** [comment]
Related to:

- https://github.com/ente-io/ente/discussions/9071
- https://github.com/ente-io/ente/discussions/10057

**Comment by abdullahkhansfc-ai** [comment]
This is just a must feature!
